trackbasket_be/models/volunteer.py

```diff
@@ -5,4 +5,5 @@ class Volunteer(BaseModel, db.Model):
   __tablename__ = 'volunteers'
   id = db.Column(db.Integer, primary_key = True)
   name = db.Column(db.String)
+  volunteer_id = db.Column(db.String)
```

Comment:
added the `volunteer_id` column to the `volunteers` table

Reply:
i enjoy your thought process behind this decision